Classes/Port/Service/LinkMappingService.php

```diff
@@ -122,7 +122,16 @@ protected function updatePropertiesWithNewRelationMapping(array $properties, str
             foreach ($this->getPropertiesWithRelations()[$tableName] as $configuration) {
                 $field = $configuration['field'];
                 $table = $configuration['table'];
+                $conditions = $configuration['conditions'];
```

Comment:
What if key 'conditions' does not exist? This could lead to a warning?

Reply:
Didn't check the logs, but `field` or `table` could also be not existing.
What about `$configuration['conditions'] ?? []`?

Reply:
Yes correct,
But field and table is always used in the examples at https://github.com/einpraegsam/migration/blob/master/Configuration/Port.php (but you are right - this could also be checked). 
While condition is set in some rare cases.
I like the `?? |]` solution :)

Reply:
Added to latest commit